src/XeroPHP/Application.php

```diff
@@ -462,14 +462,14 @@ public function delete(Remote\Model $object)
         return $object;
     }
 
-    public function updateAppRateLimit(int $appMinLimitRemining)
+    public function updateAppRateLimit($appMinLimitRemining)
```

Comment:
You reckon you could add the phpdoc version of the prop types? Just so there is no loss in the IDE auto complete

/*
* @param int $appMinLimitRemining.
*/